.circleci/config.yml

```diff
@@ -776,6 +776,29 @@ jobs:
                     echo "ERROR: Found a branch with the same name in contest-test-conf conf.json - $UNDERSCORE_CIRCLE_BRANCH.\n Merge it in order to merge the current branch into content repo."
                     exit 1
                   fi
+      - run:
+          name: Validate landingPageSections.json
+          when: always
+          command: |
+            if [[ $CIRCLE_BRANCH =~ pull/[0-9]+ ]]
+              then
+                echo "Skipping - not running in contribution build"
+                exit 0
+            fi
+            GCS_PATH=$(mktemp)
+            echo "$GCS_MARKET_KEY" > "$GCS_PATH"
+            gcloud auth activate-service-account --key-file="$GCS_PATH" > auth.out 2>&1
```

Comment:
there is no need to authenticate to download the index. the index file is public and then you can also run this on a fork. I actually expect to receive PRs from forks (marketing).